README.md

```diff
@@ -6,8 +6,13 @@ This role helps install Replicated and provides install automation for applicati
 Requirements
 ------------
 
-This role requires Ansible 1.9 or higher. The platform platform requirements are found
-in the metadata file.
+This role requires Ansible 1.9 or higher. The platform platform requirements are found in the metadata file.
```

Comment:
I think this should really say `Ansible 2.3` since that's the one you are testing with.